pkg/kubernetes/status.go

```diff
@@ -45,60 +61,68 @@ func Status() ([]StatusOutput, error) {
 
 	for _, lbl := range controlPlaneLabels {
 		go func(label string) {
-			p, err := ListPods(client, v1.NamespaceAll, map[string]string{
+			defer wg.Done()
+			// Query all namespaces for Dapr pods.
+			p, err := ListPodsInterface(client, map[string]string{
 				"app": label,
 			})
-			if err == nil {
-				pod := p.Items[0]
-				replicas := len(p.Items)
-				image := pod.Spec.Containers[0].Image
-				namespace := pod.GetNamespace()
-				age := age.GetAge(pod.CreationTimestamp.Time)
-				created := pod.CreationTimestamp.Format("2006-01-02 15:04.05")
-				version := image[strings.IndexAny(image, ":")+1:]
-				status := ""
-
-				// loop through all replicas and update to Running/Healthy status only if all instances are Running and Healthy
-				healthy := "False"
-				running := true
-
-				for _, p := range p.Items {
-					if p.Status.ContainerStatuses[0].State.Waiting != nil {
-						status = fmt.Sprintf("Waiting (%s)", p.Status.ContainerStatuses[0].State.Waiting.Reason)
-					} else if pod.Status.ContainerStatuses[0].State.Terminated != nil {
-						status = "Terminated"
-					}
-
-					if p.Status.ContainerStatuses[0].State.Running == nil {
-						running = false
-						break
-					}
-
-					if p.Status.ContainerStatuses[0].Ready {
-						healthy = "True"
-					}
+			if err != nil {
+				print.FailureStatusEvent(os.Stdout, "Error getting status, %s", err.Error())
```

Comment:
Since this will exit the goroutine and not cancel the entire status command execution, I recommend turning this into a warning and providing addition data about which control plane label this error is for.

```suggestion
print.WarningStatusEvent(os.Stdout, "Failed to get status for %s: %s", label, err.Error())
```